example/README.md

```diff
@@ -2,27 +2,28 @@
 
 ## Overview
 
-The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement by providers, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
+The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
 
 In the beginning, we will deploy a Rust echo service and call it with the Fluence CLI.
 Then we will start a JS/TS client with echo service and run it.
 
-Finally, we will use Registry to call different types of services with exactly one piece of code.
+Secondly, we will use Registry to call different types of services with exactly one piece of code.
```

Comment:
i only see the EchoService. what other services are you calling?

Reply:
Will rephase, I meant js and rust services